modules/ximgproc/include/opencv2/ximgproc/globalmatting.hpp

```diff
@@ -0,0 +1,109 @@
+#ifndef GLOBAL_MATTING_H
+#define GLOBAL_MATTING_H
+
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/ximgproc/edge_filter.hpp>
```

Comment:
Include headers which are required to describe this public API only. Remove other.